setup.py

```diff
@@ -12,31 +12,11 @@ def version():
                 return ast.parse(line).body[0].value.s
 
 
-def get_long_description():
-    with io.open("readme.md", encoding="utf-8") as f:
-        long_description = f.read()
-
-    try:
-        import pypandoc
-    except ImportError:
-        print("pypandoc not installed, using file contents.")
-        return long_description
-
-    try:
-        long_description = pypandoc.convert("readme.md", "rst")
-    except OSError:
-        print("Pandoc not found. Long_description conversion failure.")
-        return long_description
-    else:
-        long_description = long_description.replace("\r", "")
-    return long_description
-
-
 setup(
     name="curtsies",
     version=version(),
     description="Curses-like terminal wrapper, with colored strings!",
-    long_description=get_long_description(),
+    long_description=open("readme.md").read(),
```

Comment:
To make this work on Windows we should keep the `encoding='utf-8'` argument in the `open()` call.